<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUICreator</class>
 <widget class="QMainWindow" name="QUICreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit_2">
      <property name="maximumSize">
       <size>
        <width>1600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bahnschrift SemiBold'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>编译</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_11"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>皮肤</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_9"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_B"/>
   <addaction name="menu_H"/>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>关闭退出</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>词法分析</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>深沉黑</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>语法分析</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>语义分析</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>汇编代码</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset>
     <normaloff>other/image/btn_ok.png</normaloff>other/image/btn_ok.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>清浅蓝</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>简洁白</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>另存文件</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>生成可执行文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
